images/installer/nested-libvirt/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This Dockerfile is a used by CI to test a libvirt cluster launched in a gce instance
+# It builds an image containing google-cloud-sdk, ns_wrapper and scripts to launch a VM for a libvirt install.
+FROM centos:7
+
+# Enable google-cloud-sdk repo
+COPY ./images/installer/nested-libvirt/google-cloud-sdk.repo /etc/yum.repos.d/google-cloud-sdk.repo
+
+RUN yum install -y \
+    epel-release \
+    gettext \
+    google-cloud-sdk \
+    openssh-clients && \
+    yum --enablerepo=epel-testing install -y nss_wrapper && \
+    yum -y update && \
+    yum clean all
+
+COPY ./scripts/nested-libvirt/router-check.sh /router-check.sh
+COPY ./images/installer/nested-libvirt/generate-user.sh /generate-user.sh
```

Comment:
Can we put these under `/bin`?  That would match our [existing pattern][1].

[1]: https://github.com/openshift/installer/blob/e64a43d293f594c1d51a317822aa6b4783295ecb/images/installer/Dockerfile.ci#L11-L12

Reply:
yes, testing with that now :+1: